src/pages/user/authority/AuthorityCard.tsx

```diff
@@ -9,17 +9,19 @@ import { getAlmaPublication } from '../../../api/almaApi';
 import { Authority } from '../../../types/authority.types';
 import { AlmaPublication } from '../../../types/publication.types';
 import NormalText from '../../../components/NormalText';
+import useFetchUnitHierarchy from '../../../utils/hooks/useFetchUnitHierarchy';
+import AffiliationHierarchy from '../../../components/institution/AffiliationHierarchy';
 
 const StyledBoxContent = styled.div<{ isConnected: boolean }>`
   display: grid;
-  grid-template-columns: 2fr 2fr;
+  grid-template-columns: 1fr 2fr 2fr;
```

Comment:
Går an å sette `2fr 3fr 3fr` hvis du føler første kolonnen blir litt for stor. Nå synes jeg den er for liten, siden det ikke blir plass til et navn på linje med radioknappen